support/doc/plugins/guide.md

```diff
@@ -752,8 +752,7 @@ Left menu links can be filtered (add/remove a section or add/remove links) using
 
 ### Publishing
 
-PeerTube plugins and themes should be published on [NPM](https://www.npmjs.com/) so that PeerTube indexes
-take into account your plugin (after ~ 1 day). An official PeerTube index is available on https://packages.joinpeertube.org/ (it's just a REST API, so don't expect a beautiful website).
+PeerTube plugins and themes should be published on [NPM](https://www.npmjs.com/) so that PeerTube indexes take into account your plugin (after ~ 1 day).
```

Comment:
```suggestion
PeerTube plugins and themes should be published on [NPM](https://www.npmjs.com/) so that PeerTube indexes take into account your plugin (after ~ 1 day). An official plugin index is available on [packages.joinpeertube.org](https://packages.joinpeertube.org/api/v1/plugins), with no interface to present packages since its only use is [within the configuration](https://github.com/Chocobozzz/PeerTube/blob/develop/config/default.yaml#L188-L194).
```